packages/primitives/radio/meta.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import { TComponentConfig } from 'autoprops'
+import { component, startWithType } from 'refun' // mapState,
+import { RadioInput, RadioGroup, TRadioInput, TRadioGroup } from './src'
+
+export const Component = component(
+  startWithType<TRadioInput & TRadioGroup>()
```

Comment:
it could be simplified to:

```
export const Component: FC<TRadioInput & TRadioGroup> = ({ initialValue, ...radioInputProps }) => (
)
```

if `startWithType` is the only thing in the `component` pipe